ddtrace/_monkey.py

```diff
@@ -213,6 +214,15 @@ def patch_iast(**patch_modules):
             )
 
 
+def patch_subprocess_executions():
```

Comment:
rename to `patch_appsec_subprocess_executions`, WDYT?

Reply:
It's in an appsec module but yeah, more explicit for that like of code :+1: